doc/cookbook/current-route/CurrentRoute.lhs

```diff
@@ -0,0 +1,391 @@
+# How to get current route in servant-server?
+
+This tries to be an answer to the [StackOverflow question](https://stackoverflow.com/questions/41538438/haskell-servant-get-current-route-url-from-handler).
```

Comment:
I'd suggest s/the/this/ and including the "this" in the link

Reply:
Just external opinion, but consider removing the link altogether. Links are temporary, while the issue you're addressing will be around for much longer. Give a brief description instead.